AudibleApi/Api.Library.cs

```diff
@@ -282,7 +282,7 @@ public async Task<AudibleApiDTOs.ContentMetadata> GetLibraryBookMetadataAsync(st
 		#endregion
 
 		#region GetAllLibraryItemsAsync
-		public async Task<List<AudibleApiDTOs.Item>> GetAllLibraryItemsAsync()
```

Comment:
could we have both overloads? public Task<List<AudibleApiDTOs.Item>> GetAllLibraryItemsAsync() =>  GetAllLibraryItemsAsync(LibraryOptions.ResponseGroupOptions.ALL_OPTIONS);

Reply:
Added